### PR TITLE
Bulk data release notes 40

### PR DESCRIPTION
**Pull request by pmb59**

### CODE REVIEW

**Review by sfexova — APPROVED** [review]

File: release-notes/gxa/_posts/2024-05-10-40.md
Comment (ke4) [inline release-notes/gxa/_posts/2024-05-10-40.md:1]: We can add somewhere the added new publication, too:
[Expression Atlas update: insights from sequencing data at both bulk and single cell level](https://academic.oup.com/nar/article/52/D1/D107/7442051)

Comment (ke4) [inline release-notes/gxa/_posts/2024-05-10-40.md:1]: There were also some bugfix related some experiments's Supplementary Information tabs. I am not sure if we list here the fixed bugs.

**Review by ke4 — CHANGES_REQUESTED** [review]
I added 2 comments. Could you please have a look at them?
Other than that it looks fine with me.
Thanks

File: release-notes/gxa/_posts/2024-05-10-40.md
Comment (sfexova) [reply release-notes/gxa/_posts/2024-05-10-40.md:1]: Hi @ke4 - usually we wouldn't specifically mention bug fixes in the release notes, just focusing on what's new, so unless this is a bug for which we received a lot of complaints and would therefore be highly anticipated by the users I would not mention it
As for the new publication - we tweeted about the paper when it came out (which was some time ago) I think, not sure if I would mention it in the release as the paper itself is not that new if you see what I mean, we've just added it to the 'cite us' section now

**Pull request merged**